tests/PhpCompatibility/Schema071Test.php

```diff
@@ -38,7 +38,7 @@ protected function calculateParam(): mixed
     #[Test]
     public function it_is_compatible_with_phps_encoding()
     {
-        static::markTestSkipped('Literal document seems about right - yet php soap uses the type instead of the part name. Not sure what to do here yet.');
+        static::markTestSkipped('Literal document seems about right - it seems to be taking the param name instead of the complextype name though. Not sure what to do here yet.');
```

Comment:
TODO : Check spec for literal encoding on what to do here.
Current assumption: 
- if the type is a complex type, the parameter name is being used (unqualified)
- if the type is an element type, the element info is being used (qualified)

Reply:
Cant seem to figure out why it is encoding it like this. SOAP-UI is also using the message part name. Leaving it for now.